Contribute/validation-reference/gitinfonotfound.md

```diff
@@ -0,0 +1,23 @@
+---
+title: GitInfoNotFound
+description: Explanation and resolution for Docs build issue GitInfoNotFound.
+author: meganbradley
+ms.author: mbradley
+ms.topic: error-reference
+ms.date: 12/12/2018
+ms.prod: non-product-specific
+---
+# GitInfoNotFound
+
+## Warning
+
+`Author {value} cannot be recognized in GitHub.`
+
+The value of the `author` metadata attribute must be a valid GitHub ID.
```

Comment:
Not sure if it would be a separate rule, but I've come across a few docs where the `ms.author` was the same as `author`, but the `author` wasn't the same MS employee. Is validating the org membership a separate rule for later?

Reply:
We do have a backlog item to validate that the author is a member of the GitHub org that contains the repo, so that will likely catch cases of someone accidentally specifying a GitHub ID that's valid but not theirs.

We don't have any requirement for author and ms.author to be the same person, however - sometimes a non-Microsoft employee like an MVP can be the author, in which case an internal person has to be the ms.author.

Reply:
👍 Makes sense